a3fe/tests/__init__.py

```diff
@@ -14,12 +14,7 @@
 GROMACS_PRESENT = False if shutil.which("gmx") is None else True
 
 # See if the user wants to run the slurm tests
-RUN_SLURM_TESTS = True
-if os.environ.get("RUN_SLURM_TESTS") == "False":
-    RUN_SLURM_TESTS = False
-elif os.environ.get("RUN_SLURM_TESTS") is None:
-    # Default to not running SLURM tests unless explicitly enabled
-    RUN_SLURM_TESTS = False
+RUN_SLURM_TESTS = os.environ.get("RUN_SLURM_TESTS") not in [None, "False"]
```

Comment:
Ah, should probably add 0 to this list (as in `RUN_SLURM_TESTS=0` should avoid running the tests, and `RUN_SLURM_TESTS=1` should run them)! Sorry I forgot